### ISSUE TITLE
Charlga Razorflank of the Razorfen tribe and Scourge

### ISSUE THREAD

**Issue by Mechafromthe80s** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#1027]
**Describe your suggestion in full detail below:**

The suggestion is about adding a quilboar character named Charlga Razorflank (of the Razorfen tribe) to the game, and possibly adding a specific event for Razorfen tribe.
In CK2: GoA1 she is not present in the game. The ruler of the Razorflank House (tribe) is a character named Zaljazk.

I assume it would be more lore friendly to change that in the future mod for these reasons:

1) Charlga seems to have some weight behind her name. She is presented in such games as Heartstone and Warcraft Arclight Rumble, not to mention Wow itself. Not only she was present in the vanilla Wow but also in Cataclysm in "Agamaggan's Charge" quest. So it can be said she is somewhat of noteworthy character.

2) Her character plays a part in a smaller story of Scourge forces in Kalimdor. In Ultimate Visual Guide book (both versions) she is mentioned as a leader of Death's Head tribe - an undead quilboar tribe that is allied to the Razorfen tribe. However, in reality, the true leader is lich called Amnennar the Coldbringer.
However, the story of this tribe is not very clear. Details can be found here - https://wowpedia.fandom.com/wiki/Death%27s_Head_tribe
The point is - the Razorfen tribe (or more specifically Charlga) can have a unique event that would be about deciding whether her tribe joins the Scourge or rejects it. Both cases seem lore friendly if you base it on details of their alliance in WoW.

In short, adding her in the mod and expanding quilboar mechanics can help to bring more variety to the Kalimdor and lore. Back in CK2: GoA1 the part where quilboars lived was all about centaurs or taurens. Poor boars had a rough time, as usual.

That's the stuff

**Comment by VespertinePetrichor** [comment]
Thanks for your submission! This will be addressed in #1014. Linking the issue and closing this for now.
